distributed/tests/test_worker.py

```diff
@@ -217,8 +217,9 @@ def g():
     result = await future
     assert result == 123
 
-    await c.close()
     await s.close()
+    await c.close()
+
```

Comment:
Is swapping the order of these two lines meaningful?

Reply:
Ah yes, I should have made a note.

It seems when the client is closed before scheduler now, the local directory / file (foobar.py in this case) still exists, but only for a very short time. Adding a small sleep here also does the trick but swapping the order seems less 'bad'. :sweat_smile: